airflow/models.py

```diff
@@ -4359,12 +4359,13 @@ def is_backfill(self):
     @provide_session
     def get_latest_runs(cls, session):
         """Returns the latest running DagRun for each DAG. """
+        dag_states=[State.RUNNING, State.SUCCESS, State.FAILED]
```

Comment:
i have tried correcting the commit message as well, please review my pr once again and let me know if it needs any more changes. thanks.